### DISCUSSION TITLE
2020 Hyundai Sonata  recovery mode

### DISCUSSION THREAD

**Discussion by characharm** [rgerganov/gen5fw#16] [Q&A]
Hello! Can anyone please tell me how to boot into recovery mode on a 2020 Hyundai Sonata? Pressing the HOME+MAP buttons doesn't seem to have any effect. Could it be because the MAP button is touch-sensitive?

**Comment by al3emsy** [comment]
Hello, I am not sure if that is the case, but for me HOME+MAP did not work, so what I did was I first turned the car on so I don't drain the battery, then I tried POWER+(every button there is), the way I did this was by holding the two buttons then using a needle hitting the reset, I kept trying until I got the right indication which in my case was POWER+TRACK, as explained by the author, first make sure you have the boot menu before plugging the USB just incase, press on the two combinations and press the reset button, if it fails press on the POWER + the next button until you get your combination, hope this helps and goodluck!

**Comment by bogdankravchenko** [comment]
Your sonata 2020 has different sound system. This approach only for gen5 radio
